<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZonesEdit</class>
 <widget class="QWidget" name="ZonesEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Zones</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>661</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="physicalZones">
       <attribute name="title">
        <string>Physical Zones</string>
       </attribute>
       <widget class="QListWidget" name="physicalZonesList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="phAdd">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>470</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/icons/plus);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="phRemove">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>470</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/icons/minus);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="logicalZones">
       <attribute name="title">
        <string>Logical Zones</string>
       </attribute>
       <widget class="QPushButton" name="logAdd">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>470</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/icons/plus);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="logRemove">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>470</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/icons/minus);
background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QListWidget" name="logicalZonesList">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
